apps/ledger-live-mobile/src/screens/ImportAccounts/Scan.tsx

```diff
@@ -75,6 +77,13 @@ class Scan extends PureComponent<
           }
         }
       } catch (e) {
+        if (data.match(/host=([0-9A-Fa-f]+)/)) {
+          this.setState({
+            error: new ScannedNewImportQrCode(),
+            progress: 0,
+          });
+          screen("", AnalyticsPage.ScannedIncompatibleApps, { source: "Account Import Sync" });
+        }
```

Comment:
Do we have to do it in the catch block? Couldn't we perform the check at the beginning instead?

Reply:
It's the analytics event for the error that is triggered in the catch (the `setState({ error: new ScannedNewImportQrCode() })` ).
As it's handled by the generic error component I had to trigger the analytics here